README.md

```diff
@@ -39,6 +39,7 @@ Inoltre, l’allegato H del DPCM 12 ottobre al punto 2, denominato *VERIFICA
 | -----------    | ----------- | -----------        | -----------      | 
 | herald-si/verificac19-sdk-php| PHP| https://github.com/herald-si/verificac19-sdk-php | 18/11/2021 |
 | ZConnectVerificaC19SDK | IOS native (swift) | https://github.com/hrzucchetti/zconnect-verificaC19-sdk-ios | 07/12/2021 | 
+| verificac19-sdk   | Node.js | https://github.com/italia/verificac19-sdk      | 08/12/2021    | 
```

Comment:
@astagi sarebbe possibile inserire questa nuova entry come ultima riga? Giusto per rispettare l'ordine di inserimento in tabella, grazie.